src/AppInstallerCommonCore/Public/winget/ManifestLocalization.h

```diff
@@ -38,6 +41,12 @@ namespace AppInstaller::Manifest
         string_t AgreementUrl;
     };
 
+    struct Documentation
+    {
+        string_t DocumentDescription;

```

Comment:
>DocumentDescription [](http://example.com/codeflow?start=17&length=19)

nit: DocumentLabel better fits the entry purpose

Reply:
Changed to DocumentLabel